CONTRIBUTING.md

```diff
@@ -49,6 +49,20 @@ on how to start working on Servo.
 
 For specific git instructions, see [GitHub workflow 101](https://github.com/servo/servo/wiki/Github-workflow).
 
+## Running tests in pull requests
+
+When you push to a pull request, GitHub automatically checks that your changes have no compile, lint, or tidy errors.
+
+To run unit tests or Web Platform Tests against a pull request, add one or more of the labels below to your pull request:
```

Comment:
Only those with write access to the repository can add labels, so it might be worth adding sentence here explaining what to do if you cannot add a label.

```suggestion
To run unit tests or Web Platform Tests against a pull request, add one or more of the labels below to your pull request. If you do not have permission to add labels to your pull request, add a comment on your bug requesting that they be added.
```

Reply:
done in 6065670293c4992ccc0661c1a245ea84e5b3daa8